### PR TITLE
Adds zavod alt clothes for warden v2

### PR DESCRIPTION
**Pull request by LoreCoder**
Same as #15182, but patched from branch rather than master (my bad) and also removes the OG warden hat, which had snuck into the loadout options.

### CODE REVIEW

**Comment by LoreCoder** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]

File: html/changelogs/nope-wardencoat.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.
+author: Nope63/LoreCoder
```
[the remark this answers is not in the document]
Comment (alsoandanswer) [reply html/changelogs/nope-wardencoat.yml:30]: If you got an alias you can just use brackets.
For me I use Wowzewow (Wezzy).
I'd suggest using Githubname (Servername)

File: code/modules/client/preference_setup/loadout/loadout_factions.dm [hunk 2]
```diff
@@ -175,6 +175,8 @@
 	var/list/zavod_sec_coat = list()
 	zavod_sec_coat["zavodskoi security coat"] = /obj/item/clothing/suit/storage/security/officer/zav
 	zavod_sec_coat["zavodskoi security coat, alt"] = /obj/item/clothing/suit/storage/security/officer/zav/alt
+	zavod_sec_coat["zavodskoi warden coat"] = /obj/item/clothing/suit/storage/toggle/warden/zavod
+	zavod_sec_coat["zavodskoi warden coat, alt"] = /obj/item/clothing/suit/storage/toggle/warden/zavod/alt
```
[the remark this answers is not in the document]
Comment (LoreCoder) [reply code/modules/client/preference_setup/loadout/loadout_factions.dm:179]: Unsure of how to job restrict this besides adding another loadout button for it

File: html/changelogs/nope-wardencoat.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (LoreCoder) [reply html/changelogs/nope-wardencoat.yml]: Removed LoreCoder since I don't use my git name anywhere else.

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**